src/themes/standard/theme.c

```diff
@@ -699,14 +698,12 @@ GtkWindow* create_notification(UrlClickedCb url_clicked)
 	atk_object_set_description(atkobj, "Notification summary text.");
 
 	/* Add the close button */
-	alignment = gtk_alignment_new(1, 0, 0, 0);
-	gtk_widget_show(alignment);
-	gtk_box_pack_start(GTK_BOX(hbox), alignment, FALSE, FALSE, 0);
-
 	close_button = gtk_button_new();
+	gtk_widget_set_halign (close_button, GTK_ALIGN_END);
```

Comment:
Should we also set valign to "start" here to keep it the same as with gtk_alignment_new...?